Makefile

```diff
@@ -121,6 +121,18 @@ uninstall_udev:
 	$(if $(UDEV_RULES_DIR),,$(error UDEV_RULES_DIR is not defined))
 	rm -f ${DESTDIR}/${UDEV_RULES_DIR}/40-monitor-hotplug.rules
 
+# Rules for manpage
+MANPAGES_DIR:=/usr/share/man/man1
```

Comment:
This variable is typically called `MANDIR`, and it should be defined relative to `PREFIX`, that is `${PREFIX}/share/man/man1`.